docs/resources/saml_group_mappings.md

```diff
@@ -0,0 +1,85 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "lightstep_saml_group_mappings Resource - terraform-provider-lightstep"
+subcategory: ""
+description: |-
+  Provides a Lightstep SAML Group Mapping to automatically update user's roles based on their SAML attributes. For conceptual information about managing SAML group mappings, visit Lightstep's documentation https://docs.lightstep.com/docs/TBD.
+---
+
+# lightstep_saml_group_mappings (Resource)
+
+Provides a Lightstep SAML Group Mapping to automatically update user's roles based on their SAML attributes. For conceptual information about managing SAML group mappings, visit [Lightstep's documentation](https://docs.lightstep.com/docs/TBD).
+
+## Example Usage
+
+```terraform
+resource "lightstep_saml_group_mappings" "group_mappings" {
+  mapping {
+    # For users with the "member_of: sre" SAML Attribute,
+    # Assign "Organization Editor" in the organization.
+    match {
+      attribute_key = "member_of"
+      attribute_value = "sre"
+    }
+    roles {
+      organization_role = "Organization Editor"
+    }
+  }
+
+  mapping {
+    # For users with the "member_of: developer" SAML Attribute,
+    # Assign "Organization Restricted Member" in the organization, and
+    # Assign "Project Viewer" in the "Project A" project.
+
+    match {
+      attribute_key = "member_of"
+      attribute_value = "developer"
+    }
+    roles {
+      organization_role = "Organization Restricted Member"
+      project_roles = {
+            "Project A" =  "Project Viewer"
+      }
+    }
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `mapping` (Block List, Min: 1) List of SAML Group Mappings. (see [below for nested schema](#nestedblock--mapping))
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
+
+<a id="nestedblock--mapping"></a>
+### Nested Schema for `mapping`
+
+Required:
+
+- `match` (Block List, Min: 1, Max: 1) Key/Value attribute pair to match against the user's SAML attributes. (see [below for nested schema](#nestedblock--mapping--match))
+- `roles` (Block List, Min: 1, Max: 1) Roles to assign to the user if the match is successful. (see [below for nested schema](#nestedblock--mapping--roles))
+
+<a id="nestedblock--mapping--match"></a>
+### Nested Schema for `mapping.match`
+
+Required:
+
+- `attribute_key` (String) Attribute Key to match against the user's SAML attributes.
+- `attribute_value` (String) Attribute Value to match against the user's SAML attributes
+
+
+<a id="nestedblock--mapping--roles"></a>
+### Nested Schema for `mapping.roles`
+
+Required:
+
+- `organization_role` (String) Organization Role. Only 'Organization Editor' and 'Organization Viewer' are supported.
```

Comment:
Should Organization Restricted Member be a valid option in this list as well?

Reply:
yeah, nice catch! Updated it on both places, thank you!